cfme/tests/configure/test_access_control.py

```diff
@@ -128,6 +128,31 @@ def test_user_assign_multiple_groups(appliance, request, group_collection):
         "User's assigned groups are different from expected groups")
 
 
+@pytest.mark.tier(2)
+@pytest.mark.uncollectif(lambda appliance: appliance.version < '5.9')
+def test_user_change_groups(appliance, request, group_collection):
+    """Assign a user to multiple groups and confirm that the user can successfully change groups
+    """
+    group_names = [
+        'EvmGroup-user', 'EvmGroup-user_self_service', 'EvmGroup-desktop']
+
+    group_list = [group_collection.instantiate(description=names) for names in group_names]
+
+    user = new_user(appliance, groups=group_list)
+    request.addfinalizer(user.delete)
+    request.addfinalizer(user.appliance.server.login_admin)
+
+    view = appliance.server.login(user)
+
+    curr_group = view.current_groupname
+
+    for group in [name for name in group_names if name != curr_group]:
+        view.change_group(group)
+        assert group == view.current_groupname, (
+            "User failed to change current group from {} to {}".format(
+                view.current_groupname, group))
+
+
```

Comment:
**QUESTION:** Do we want to add a change back to the original group (which is held in curr_group presently).

Reply:
I don't think so since we creating a user specifically for this test and deleting it at the end.

Reply:
It's not to leave the system in a sane state, its to make sure nothing weird happens when you change back to the same group (i.e. for completeness).

Reply:
That's a valid point and I've updated the test so that it cycles back around to the original group. 
This test was designed to test basic webui functionality that the user can change groups. A true test for completeness would be implemented in a future test that would verify the users ability to iterate over all the possible permutations of default group transitions while validating that the navigation ui is updated accordingly after each transition.